frontend/packages/dev-console/src/components/topology/shapes/PodSet.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { get } from 'lodash';
+import {
+  PodStatus,
+  calculateRadius,
+  getDataBasedOnCurrentAndPreviousRC,
+  rollback,
+} from '@console/shared';
+import { k8sUpdate } from '@console/internal/module/k8s';
+import { DeploymentConfigModel } from '@console/internal/models';
+import { DonutStatusData } from '../topology-types';
+
+interface PodSetProps {
+  size: number;
+  data: DonutStatusData;
+}
+
+const calculateInnerPodStatusRadius = (
+  outerPodStatusInnerRadius: number,
+  outerPodStatusWidth: number,
+): { innerPodStatusOuterRadius: number; innerPodStatusInnerRadius: number } => {
```

Comment:
nit: create a type for this so that it can be used in other places as well if needed.

Reply:
fixed in https://github.com/openshift/console/pull/2456/commits/d069d28b40bb578d9a26810bb61269647f5d241b